pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, mkDerivation, fetchurl, makeFontsConf, appimageTools,
+  qtbase, qtsvg, qtmultimedia, qtwebsockets, qtimageformats,
+  autoPatchelfHook, desktop-file-utils, imagemagick, makeWrapper,
+  twemoji-color-font, xorg, libsodium, libopus, libGL, zlib, alsaLib }:
+
+mkDerivation rec {
+  pname = "ripcord";
+  version = "0.4.23";
+
+  src = let
+    appimage = fetchurl {
+      url = "https://cancel.fm/dl/Ripcord-${version}-x86_64.AppImage";
+      sha256 = "0395w0pwr1cz8ichcbyrsscmm2p7srgjk4vkqvqgwyx41prm0x2h";
+      name = "${pname}-${version}.AppImage";
+    };
+  in appimageTools.extractType2 {
+    inherit name;
```

Comment:
`name` doesn't exist anymore.

Reply:
ah, I apologize, 
```suggestion
    name = "${pname}-${version}";
```

Reply:
My bad. I updated that without even checking it